rasa/shared/data.py

```diff
@@ -166,48 +149,6 @@ def is_nlu_file(file_path: Text) -> bool:
     return nlu_loading.guess_format(file_path) != nlu_loading.UNK
 
 
-def is_story_file(file_path: Text) -> bool:
-    """Checks if a file is a Rasa story file.
-
-    Args:
-        file_path: Path of the file which should be checked.
-
-    Returns:
-        `True` if it's a story file, otherwise `False`.
-    """
-    from rasa.shared.core.training_data.story_reader.yaml_story_reader import (
-        YAMLStoryReader,
-    )
-    from rasa.shared.core.training_data.story_reader.markdown_story_reader import (
-        MarkdownStoryReader,
-    )
-
-    return YAMLStoryReader.is_stories_file(
-        file_path
-    ) or MarkdownStoryReader.is_stories_file(file_path)
-
-
-def is_test_stories_file(file_path: Text) -> bool:
-    """Checks if a file is a test stories file.
```

Comment:
Similar question for both function `is_test_stories_file` and `is_story_file` - why were these removed? Might have missed a conversation here 😅

Reply:
Because they were literally just proxy methods and were calling one function within themselves.